app/src/main/java/com/orange/ouds/app/ui/tokens/TokenProperty.kt

```diff
@@ -176,7 +176,7 @@ sealed class TokenProperty<T>(
     )
 
     data object SpaceRowGap : TokenProperty<TokenCategory.Dimension.Space>(
-        nameRes = R.string.app_tokens_dimension_space_rowGap_label,
+        nameRes = R.string.app_tokens_dimension_space_gapStack_label,
```

Comment:
I find it confusing that the documentation talks about `gap stack` whereas the associated tokens are named `row-gap`. Do we know why it's different?

Reply:
I think this is a mistake, I thought that "gap stack" was the last naming.